etc/adminhtml/system.xml

```diff
@@ -717,6 +721,151 @@
+                        <source_model>Magento\Directory\Model\Config\Source\Country</source_model>
+                        <depends>
+                            <field id="allowspecific">1</field>
+                        </depends>
+                    </field>
+                </group>
+                <group id="omise_offsite_duitnowqr" translate="label" sortOrder="523" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>Enable DuitNow QR Payment</label>
+                    <comment>Enable customers to pay using DuitNow QR payment method</comment>
+                    <field id="active" translate="label comment" type="select" sortOrder="524" showInDefault="1" showInWebsite="1" showInStore="1">
+                        <label>Enable/Disable</label>
+                        <comment>Enable DuitNow QR payment method.</comment>
+                        <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+                        <config_path>payment/omise_offsite_duitnowqr/active</config_path>
+                    </field>
+                    <field id="title" translate="label" type="text" sortOrder="525" showInDefault="1" showInWebsite="1" showInStore="1">
+                        <label>Title</label>
+                        <comment>This controls the title which the user sees during checkout.</comment>
+                        <config_path>payment/omise_offsite_duitnowqr/title</config_path>
+                    </field>
+                    <field id="allowspecific" translate="label comment" type="select" sortOrder="526" showInDefault="1" showInWebsite="1" showInStore="1">
+                        <label>Allowed Countries</label>
+                        <config_path>payment/omise_offsite_duitnowqr/allowspecific</config_path>
+                        <source_model>Magento\Payment\Model\Config\Source\Allspecificcountries</source_model>
+                        <comment>If not set to all, guest customers will not have a billing country and may not be able to checkout.</comment>
+                    </field>
+                    <field id="specificcountry" translate="label" type="multiselect" sortOrder="527" showInDefault="1" showInWebsite="1" showInStore="1">
+                        <label>Payment from Specific countries</label>
+                        <config_path>payment/omise_offsite_duitnowqr/specificcountry</config_path>
+                        <source_model>Magento\Directory\Model\Config\Source\Country</source_model>
+                        <depends>
+                            <field id="allowspecific">1</field>
+                        </depends>
+                    </field>
+                </group>
+                <group id="omise_offsite_maybankqr" translate="label" sortOrder="528" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>Enable Maybank QRPay Payment</label>
```

Comment:
Should it be `Maybank QRPAY` ?

Reply:
Nice catch here updated -> https://github.com/omise/omise-magento/pull/366/commits/33576558e3d4d071dafe05722a62152cb58e7de3 🙏